source/auth/auth.rst

```diff
@@ -1102,44 +1101,6 @@ As a URI, those have to be UTF-8 encoded and URL-escaped, e.g.:
 - mongodb://%E2%85%A8:IV@mongodb.example.com/admin
 - mongodb://%E2%85%A8:I%C2%ADV@mongodb.example.com/admin
 
---------------------------
-Speculative Authentication
---------------------------
```

Comment:
I think it would be nice to keep this section and simply link to the handshake spec.

Reply:
Yeah, that matches what you did when you split SDAM, which worked quite nicely.  SGTM.

Reply:
I have added the link to the handshake spec.